contrib/natss/pkg/dispatcher/dispatcher.go

```diff
@@ -211,21 +213,34 @@ func (s *SubscriptionsSupervisor) UpdateSubscriptions(channel *eventingv1alpha1.
 		chMap = make(map[subscriptionReference]*stan.Subscription)
 		s.subscriptions[cRef] = chMap
 	}
+	var errStrings []string
 	for _, sub := range subscriptions {
+		subscriberStatus := ToSubscriberStatus(&sub, corev1.ConditionFalse, "internal error")
```

Comment:
this function needs to be exposed? I think is only used here?

Reply:
another cosmetic: seems that you don't need the condition nor the message as parms as you are only calling this here?
maybe call it `newSubscriberStatus`? and just pass the &sub.
I'm OK if you leave it as is, but unexposed